podpac/core/coordinates/uniform_coordinates1d.py

```diff
@@ -269,6 +269,14 @@ def coordinates(self):
         # coordinates.setflags(write=False)  # This breaks the 002-open-point-file example
         return coordinates
 
+    @property
+    def ndim(self):
+        return 1  # TODO ND
```

Comment:
I  can't even imagine what a multi-dimensional uniform coordinates would look like... you could probably remove the TODO.